packages/frontend/src/Build.svelte

```diff
@@ -66,6 +73,24 @@ async function fillBuildOptions() {
   }
 }
 
+async function fillArchitectures(historyInfo: BootcBuildInfo[]) {
```

Comment:
There's a lot of overlap with fillBuildOptions, which even sets buildArch in one case. It's fine as-is, but I think they should probably be merged.